conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalytics.java

```diff
@@ -291,54 +259,60 @@ private void customEvent(BitmovinPlayerEvent event, Map<String, Object> attribut
 
     // region Session handling
     private void setupPlayerStateManager() {
-        try {
-            playerStateManager = client.getPlayerStateManager();
-            playerStateManager.setPlayerState(PlayerStateManager.PlayerState.STOPPED);
-            playerStateManager.setPlayerType("Bitmovin Player Android");
-            playerStateManager.setPlayerVersion(playerHelper.getSdkVersionString());
-        } catch (ConvivaException e) {
-            Log.e(TAG, e.getLocalizedMessage());
+        convivaVideoAnalytics = com.conviva.sdk.ConvivaAnalytics.buildVideoAnalytics(context);
+        convivaVideoAnalytics.setCallback(videoAnalyticsCallback);
+        setPlayerInfo();
+    }
+
+    private void setPlayerInfo() {
+        if (convivaVideoAnalytics != null) {
+            Map<String, Object> playerInfo = new HashMap<>();
+            playerInfo.put(ConvivaSdkConstants.PLAYER_NAME, contentMetadataBuilder.getApplicationName());
+            playerInfo.put(ConvivaSdkConstants.FRAMEWORK_NAME, "Bitmovin Player Android");
+            playerInfo.put(ConvivaSdkConstants.FRAMEWORK_VERSION, playerHelper.getSdkVersionString());
+            boolean isLive = contentMetadataBuilder.getStreamType() == ConvivaSdkConstants.StreamType.LIVE;
+            playerInfo.put(ConvivaSdkConstants.IS_LIVE, isLive);
+            convivaVideoAnalytics.setPlayerInfo(playerInfo);
         }
     }
 
+    private ConvivaExperienceAnalytics.ICallback videoAnalyticsCallback = new ConvivaExperienceAnalytics.ICallback() {
+        @Override
+        public void update() {
+        }
+
+        @Override
+        public void update(String s) {
+        }
+    };
```

Comment:
Do we need this if we don't implement anything in there? 🤔

Reply:
Removed